perun-core/src/main/java/cz/metacentrum/perun/core/api/ServicesManager.java

```diff
@@ -849,13 +850,13 @@
 	 * @param destination (Id of this destination doesn't need to be filled.)
 	 * @return destination with it's id set.
 	 * @throws PrivilegeException
-	 * @throws InternalErrorException
+	 * @throws InvalidDestinationException if destination has invalid name
```

Comment:
Hmm, its not exactly "name". Its more like invalid value (since param name is same as the object itself). Maybe also specify, that invalid value depends on type.

Reply:
Ok, changed to `if destination has invalid value in given destination type`.